Ecommerce/index.css

```diff
@@ -7,4 +7,19 @@ nav{
     justify-content: center;

```

Comment:
There is no opening brace for a code block. Add an opening brace to start the block.